src/js/containers/crossFile/CrossFileContentContainer.jsx

```diff
@@ -214,9 +231,16 @@ class CrossFileContentContainer extends React.Component {
 		if(this.props.submission.publishStatus !== "unpublished") {
 			warningMessage = <PublishedSubmissionWarningBanner />;
 		}
+
+		let gtasWarning = null;
+		if(this.state.gtas && this.state.gtas.open){
```

Comment:
gtas.open no longer exists. Given the suggestion below you can probably just do the first part of this if for checking.